code/game/objects/items/melee/misc.dm

```diff
@@ -325,7 +325,7 @@
 	if(!iscarbon(user))
 		target.LAssailant = null
 	else
-		target.LAssailant = user
+		target.LAssailant = WEAKREF(user)
```

Comment:
Maybe I'm missing some other code in another file, but what is this variable for if you can replace it with a WEAKREF and change nothing else?

Reply:
![image](https://user-images.githubusercontent.com/58055496/120294578-2e63f200-c27b-11eb-8b9d-e72f10a09241.png)
behold, slime code

Reply:
yes this is literally it